Formula/unit.rb

```diff
@@ -32,6 +32,12 @@ def post_install
     (var/"run/unit").mkpath
     (var/"state/unit/certs").mkpath
   end
+  
+  plist_options manual: "unitd --no-daemon"
```

Comment:
What is the purpose of this line? It seems brew services works fine without it.

Reply:
I tried, and I can't find any effect of that. Just some core Formulas having that line before service block. Will omit that.